cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -260,14 +282,16 @@ static int _request_set_trx_state(ieee802154_dev_t *dev, ieee802154_trx_state_t
 {
 
     (void) dev;
-    bool wait_sfd = RFCORE->XREG_FSMSTAT0bits.FSM_FFCTRL_STATE >= CC2538_STATE_SFD_WAIT_RANGE_MIN
-                    && RFCORE->XREG_FSMSTAT0bits.FSM_FFCTRL_STATE <= CC2538_STATE_SFD_WAIT_RANGE_MAX;
+    bool wait_sfd = 
```

Comment:
```suggestion
    bool wait_sfd =
```

trailing whitespace - squash directly.